spec/registry.md

```diff
@@ -255,3 +255,500 @@ which only expects the `accord` option:
 > .input {$object :noun case=nominative}
 > {{You see {$color :adjective article=indefinite accord=$object} {$object}!}}
 >```
+
+# Default Registry
+
+This section describes the functions which each implementation MUST provide to be conformant with
+this specification.
+
+## String Value Selection and Formatting
+
+(Pending)
+
+## Numeric Value Selection and Formatting
+
+### Functions
+
+The function `:number` is the default selector and formatter for numeric values.
+
+The function `:integer` provides a reduced set of options for selecting
+and formatting numeric values as integers.
+
+### Operands
+
+The _operand_ of a number function is either an implementation-defined type or
+a literal that matches the `number-literal` production in the [ABNF](/spec/message.abnf).
+All other values produce a _Resolution Error_.
```

Comment:
This is inconsistent with Date and Time Value Formatting:
> All other _operand_ values produce a _Selection Error_ when evaluated for selection or a _Formatting Error_ when formatting the value.

Reply:
If we didn't correct date/time previously, I will do so. This was a discussion with @eemeli. The _Resolution Error_ is what comes first.